.gitignore

```diff
@@ -185,8 +185,8 @@ copier_scripts/
 /cms/jinja2/layout
 /cms/static_compiled
 
-node_modules/
-static/
+/node_modules/
+/static/
```

Comment:
Ideally we would keep all static files in `static_src` - we can have a separate folder for `admin` files that relate to wagtail admin if needed.

Reply:
Just to clarify: `static_src` is good for FE stuff that needs to be 'packaged up' for end users. And that should rightly never be in `.gitignore`. It's also valid for apps to have static files that are only needed for working inside Wagtail, which shouldn't be bundled in the same way. This change is to allow for that second scenario.

Reply:
Yep, but this can still be done via files that live in the `static_src` directory -example on the bucks project (with apologies to ONS devs who won't be able to follow this link): https://git.torchbox.com/buckinghamshire-council/bc/-/tree/release/bc/static_src/javascript?ref_type=heads

Reply:
@helenb Nothing about this PR is really suggesting what the end-goal approach should be (which would be one to iron out in review). Merely that, it's simply the easiest way to solve a problem a lot of the time (e.g. when there hasn't been any FE involvement yet), and ignoring those files by default doesn't make sense.

Reply:
Ok sure happy to approve.